lib/sw-toolbox.js

```diff
@@ -121,7 +40,7 @@ module.exports = {
   fastest: strategies.fastest,
   router: router,
   options: options,
-  cache: cache,
-  uncache: uncache,
-  precache: precache
+  cache: helpers.cache,
+  uncache: helpers.uncache,
+  precache: helpers.precache
 };
```

Comment:
This cleans up the main sw-toolbox entry quite a bit while making it easier to just pull in the helpers. It's also passing our current CI tests. I'm personally fine with this change, but let's see what @jeffposnick and @gauntface think.